### ISSUE TITLE
player: playback fails with TrueHD stream selected, but Direct Stream disallowed

### ISSUE THREAD

**Issue by OttoKerner** [plexinc/plex-media-player#702]
### Test environment
**PMS Version:** 1.13.0.4987
**PMP Version:** 2.9.0.824
**Platform:** Windows

### Steps to reproduce
1. have a video file with a TrueHD audio track
2. select this audio track
3. disable 'Direct Play' and 'Direct Stream' in the preferences of PMP (fullscreen mode)
4. leave the video bitrate preference at 'Original'

### Current behavior
1. Playback fails to start. Status in the web app shows that 'Direct Stream' of the audio stream is attempted, even though this is forbidden via the client preference.

### Expected behavior
1. Audio should get transcoded and playback should start

**Comment by LongChair** [comment]
ping @markus101

**Comment by OttoKerner** [comment]
The user who reported this is on FreeBSD. https://forums.plex.tv/discussion/comment/1657449/#Comment_1657449
He says TrueHD transcoding doesn't work at all for him.

I don't know if this is a different issue or if these issues have a common root.

**Comment by mseeley** [comment]
The 'Direct Play' and 'Direct Stream' settings are part of the video section. They don't apply to audio streams. It makes sense that one would want to disable direct play of an audio stream as a workaround  but currently this is unsupported.

I was able to successfully DP a video with TrueHD audio in a Windows 10 VM. The same video played with audio after I unchecked 'Direct Play' and 'Direct Stream'. DP of the same video on macOS also resulted in correct playback.

> The user who reported this is on FreeBSD. https://forums.plex.tv/discussion/comment/1657449/#Comment_1657449
> He says TrueHD transcoding doesn't work at all for him.
> 
> I don't know if this is a different issue or if these issues have a common root.

It could be the necessary codecs are not downloaded correctly on Linux. We don't build or officially support Linux.
